lib/broccoli/fastboot-config.js

```diff
@@ -143,11 +143,10 @@ module.exports = class FastBootConfig extends Plugin {
 
     this.dependencies = dependencies;
     this.moduleWhitelist = uniq(moduleWhitelist);
-  };
```

Comment:
Umm, how did this work before?  Is this file even used? This seems like it would have been a parser error to me?!?!?

Reply:
It seems to only be a lint problem